src/QuizAtom.tsx

```diff
@@ -1,7 +1,322 @@
+
+export const QuizAtom = ({ id, questions }: QuizAtomType): JSX.Element => (
+    <div data-atom-id={id}>
+        <form>
+            {questions.map((question) => (
+                <Question
+                    key={question.id}
+                    number={questions.indexOf(question) + 1}
+                    {...question}
+                />
+            ))}
+        </form>
+    </div>
+);
+
+export type QuestionProps = {
+    number: number;
+};
+
+export const Question = ({
+    text,
+    imageUrl,
+    answers,
+    number,
+}: QuestionType & QuestionProps): JSX.Element => {
+    const [Chosen, setChosen] = useState<string | undefined>(undefined);
+
+    return (
+        <div
+            className={css`
+                ${body.medium()}
+            `}
+        >
+            <fieldset className={fieldsetStyle}>
+                <div
+                    className={css`
+                        display: flex;
+                    `}
+                >
+                    <div
+                        className={css`
+                            padding-right: 0.5ch;
+                        `}
+                    >
+                        {number + '.'}
+                    </div>
+                    <legend
+                        className={css`
+                            margin-bottom: 12px;
+                        `}
+                    >
+                        {text}
+                    </legend>
+                </div>
+                {imageUrl && (
+                    <img className={fullWidthStyle} src={imageUrl}></img>
+                )}
+                <div
+                    className={css`
+                        display: flex;
+                        flex-direction: column;
+                    `}
+                >
+                    {answers.map((answer) => (
+                        <Answer
+                            key={answer.id}
+                            isChosen={Chosen === answer.id}
+                            {...answer}
+                            setChosen={setChosen}
+                            isAnswered={Chosen !== undefined}
+                        />
+                    ))}
+                </div>
+            </fieldset>
+        </div>
+    );
+};
+
+type AnswerProps = {
+    isChosen: boolean;
+    setChosen: React.Dispatch<React.SetStateAction<string | undefined>>;
+    isAnswered: boolean;
+};
+
+export const Answer = ({
+    id,
+    text,
+    isCorrect,
+    revealText,
+    isChosen,
+    setChosen,
+    isAnswered,
+}: AnswerType & AnswerProps): JSX.Element => (
+    <div
+        className={css`
+            clear: both;
+            margin-bottom: 6px;
+        `}
+    >
+        <input
+            type="radio"
+            id={`answer-${id}`}
+            required
+            checked={isChosen}
+            onClick={() => setChosen(id)}
+            disabled={isAnswered}
+            className={visuallyHidden}
+        ></input>
+        <label
```

Comment:
I think there's some accessibility work required to make the answers tabble-through? 

Testing on https://5f22d2148807980022fc782e-bougkfzsdn.chromatic.com/iframe.html?id=quizatom--default-story

I was seeing a couple of things:

- No focus state as tabbing through the questions, so I was not able to see the current question I was on 
- Because of the 'onClick' as soon as the arrow keys are used to navigate between the radio buttons, the answers are chosen.

I will say though, Frontend doesn't fair much better in this respect (tabbing to an answer - it has a focus state which is good, moving through radio buttons with arrow keys selects my answer):

![2020-08-26 10 35 37](https://user-images.githubusercontent.com/638051/91287710-fe397100-e787-11ea-8a6a-19b64fde02f6.gif)

But it's worth having a think to see if we can improve this in this version.

Maybe you could check the eventType:

```
if(event.type == 'mousedown')
    {
        // do something
        return;
    }
    if(event.type == 'keypress')
    {
        // do something else
        return;
    }`
```

And dealing with things slightly differently 🤔  I'm not 100% sure what this looks like though!

Reply:
Thanks Gareth, I wouldn't have thought to check this. @nicl helped me make the quiz useable without a mouse today - thank you.

![tabbable](https://user-images.githubusercontent.com/9820960/92752705-12cd5a00-f381-11ea-9c96-639cb1b48b42.gif)

Reply:
Hmm it looks like I need to work out how to make a gif!